build/ansible/pmm/systemd.yml

```diff
@@ -0,0 +1,42 @@
+---
+# This playbook contains tasks executed during PMM Server update in non-docker environments.
+# TODO: refactor from supervisord to systemd if necessary.
+- hosts: all
+  become: true
+  remote_user: root
+  gather_facts: true
+
+  # TODO: replace supervisord.service with pmm.service
+  tasks:
+    # Note: forking type must be set to 'simple'
+    - name: Configure systemd
```

Comment:
Supervisord, not systemd. Same in the file name.

Reply:
Well, I left this one for the future AMI/OVF needs. It's not used for our docker distribution. 

What you mention if fair:  it's looks like a collection of old tasks for supervisord. However, in AMI/OVF build `supervisord` will be replaced by `systemd`, since we only need to start docker/podman/k8s and deploy PMM to it.

No worries anyway, it'll be refactored in a further k8s ticket.

Reply:
confusion here is that step has name `Configure systemd` but it copies `supervisord.service` item.

Anyway, I'm fine to keep it as is if you aware about it.

Reply:
I updated the task description to avoid confusion :)